packages/@aws-cdk/aws-ec2/README.md

```diff
@@ -24,32 +24,103 @@ import ec2 = require('@aws-cdk/aws-ec2');
 const vpc = new ec2.Vpc(this, 'VPC');
 ```
 
-All default Constructs requires EC2 instances to be launched inside a VPC, so
+All default constructs require EC2 instances to be launched inside a VPC, so
 you should generally start by defining a VPC whenever you need to launch
 instances for your project.
 
-Our default `Vpc` class creates a private and public subnet for every
-availability zone. Classes that use the VPC will generally launch instances
-into all private subnets, and provide a parameter called `vpcSubnets` to
-allow you to override the placement. [Read more about
+### Subnet Types
+
+A VPC consists of one or more subnets that instances can be placed into. CDK
+distinguishes three different subnet types:
+
+* **Public** - public subnets connect directly to the Internet using an
+  Internet Gateway. If you want your instances to have a public IP address
+  and be directly reachable from the Internet, you must place them in a
+  public subnet.
+* **Private** - instances in private subnets are not directly routable from the
+  Internet, and connect out to the Internet via a NAT gateway. By default, a
+  NAT gateway is created in every public subnet for maximum availability. Be
+  aware that you will be charged for NAT gateways.
+* **Isolated** - isolated subnets do not route from or to the Internet, and
+  as such do not require NAT gateways. They can only connect to or be
+  connected to from other instances in the same VPC. A default VPC configuration
+  will not include isolated subnets,
+
+A default VPC configuration will create public and private subnets, but not
+isolated subnets. See *Advanced Subnet Configuration* below for information
+on how to change the default subnet configuration.
+
+Classes that use the VPC will "launch instances" (or more accurately, create
```

Comment:
"Classes that use the VPC" -- not clear here. Perhaps you mean 'CDK constructs'?